python/lsst/ts/MTDome/mock_llc/thcs.py

```diff
@@ -40,7 +40,7 @@ def __init__(self):
         self.log = logging.getLogger("MockThcsStatus")
         # variables holding the status of the mock Louvres
         self.status = MotionState.CLOSED
-        self.temperature = np.zeros(_NUM_SENSORS, dtype=float)
+        self.temperature = np.zeros(NUM_THERMO_SENSORS, dtype=float)
```

Comment:
The comment "# variables holding ..." (a few lines above) looks out of place here. Please consider deleting it.

Reply:
Nice catch. It was yet another case of bad copying and pasting. I removed the comments as you suggested.